### DISCUSSION TITLE
Can't sign in to dashboard?

### DISCUSSION THREAD

**Discussion by aliuq** [supabase/supabase#5728] [Questions] [answered]
![image](https://user-images.githubusercontent.com/11707934/156103627-c56d9c9b-228d-4551-a9af-4196ee6ea4f8.png)

**Comment by J0** [comment, marked as answer]
Hey,

Thanks for your query! We're sorry to hear that the dashboard isn't functioning as expected for you. Do contact support@supabase.io for further assistance!

JL
